### ISSUE TITLE
a tiny error in character_rnn example kurfile

### ISSUE THREAD

**Issue by EmbraceLife** [deepgram/kur#29]
Hi @noajshu ,
Thanks for your character example, it is great! 

@ajsyp has pointed to me something is not exactly right with the weights, and now I realized the tiny issue there.  Inside kurfile.yaml, is the following `weights` what you wanted, instead of [here](https://github.com/deepgram/kur/blob/master/examples/language-model/model/kurfile.yaml#L46): 

```yml
train:
  data:
    - jsonl: ../data/train.jsonl
  epochs: 5
  weights:
    initial: best.w.kur
    best: best.w.kur
    last: last.w.kur
  log: log


validate:
  data:
    - jsonl: ../data/validate.jsonl
  weights: best.w.kur


test:
  data:
    - jsonl: ../data/test.jsonl
  weights: best.w.kur
```

**Comment by ajsyp** [comment]
My recommendation is to do the following:
- Use `initial` and `last` keys under the training weights (but not `best`). The reason for this is that if `best` is specified under training weights, then Kur will save the best weights _with respect to the training loss_ to the file at `best`. `Best` under validation tells Kur to save the best weights _with respect to the validation loss` to the file at `best`. Thus, using _both_ puts that file in competition. So either drop `best` out of training weights entirely, or just change the name, something like `best: best.train.w.kur`.
- Use the best weights with respect to validation loss in the test and evaluate sections. This is what is shown in your examples, @EmbraceLife.

If you feel up to it, go ahead and submit a pull request for this!
